Stage 3/script.js

```diff
@@ -0,0 +1,81 @@
+const quizForm = document.getElementById("quiz-form");
+const questionList = document.getElementById("question-list");
+const player1Score = document.getElementById("player1-score");
+const player2Score = document.getElementById("player2-score");
+const winnerMessage = document.getElementById("winner-message");
+
+let questions = [];
+let player1Points = 0;
+let player2Points = 0;
+
+function addQuestion() {
+  const question = document.getElementById("question").value;
+  const option1 = document.getElementById("option1").value;
+  const option2 = document.getElementById("option2").value;
+  const option3 = document.getElementById("option3").value;
+  const option4 = document.getElementById("option4").value;
+  const correctAnswer = document.getElementById("correct-answer").value;
+
+  const newQuestion = {
+    question,
+    options: [option1, option2, option3, option4],
+    correctAnswer,
+  };
+
+  questions.push(newQuestion);
+  renderQuestions();
+  quizForm.reset();
+}
+
+function renderQuestions() {
+  questionList.innerHTML = "";
+  questions.forEach((q, index) => {
+    const listItem = document.createElement("li");
+    listItem.classList.add("question");
+    listItem.textContent = `Q${index + 1}: ${q.question}`;
+    questionList.appendChild(listItem);
+  });
+}
+
+function revealAnswers() {
```

Comment:
Think there might be a misunderstanding here.. It should reveal the correct answer of the spesific question like in your week2 implementation